src/service/impl/mobility/index.ts

```diff
@@ -220,12 +220,18 @@ export default (): IMobilityService => ({
         {headers: {'x-api-key': nivelApiKey}},
         nivelBaseUrl,
       );
-      const result = violationsReportingInitQueryResultSchema.validate(
-        response,
-        {
-          stripUnknown: true,
-        },
-      );
+      const mapped = {
+        ...response,
+        providers: response.providers.map((provider) => ({
+          ...provider,
+          // The image field can in some cases be null, but the app expects it
+          // to be either set or undefined
+          image: provider.image ?? undefined,
+        })),
+      };
+      const result = violationsReportingInitQueryResultSchema.validate(mapped, {
```

Comment:
It would have been nice to get a proper error message if there was a validation error. Right now it says Internal Server Error,  so one has to debug it locally to see what failed actually.

Reply:
True! Made an issue for doing this wherever it applies: https://github.com/AtB-AS/kundevendt/issues/18059